index.test-d.ts

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable @typescript-eslint/no-confusing-void-expression */
+
+import {expectType} from 'tsd';
+import alfy, {ScriptFilterItem} from './index.js';
+
+const mockItems: ScriptFilterItem[] = [
+	{
+		title: 'Unicorn'
+	},
+	{
+		title: 'Rainbow'
+	}
+];
+
+expectType<void>(alfy.output(mockItems));
+
+expectType<string[]>(alfy.matches('Corn', ['foo', 'unicorn']));
+
+expectType<ScriptFilterItem[]>(alfy.matches('Unicorn', mockItems, 'title'));
+
+expectType<string[]>(alfy.inputMatches(['foo', 'unicorn']));
+
+expectType<ScriptFilterItem[]>(alfy.inputMatches(mockItems, 'title'));
+
+expectType<void>(alfy.error(new Error('some error')));
+
+expectType<void>(alfy.log('some message'));
+
+expectType<Promise<any>>(alfy.fetch('https://api.foo.com', {
```

Comment:
Let's use a nonexistent URL instead.

```suggestion
expectType<Promise<any>>(alfy.fetch('https://foo.bar', {
```